### DISCUSSION TITLE
Nominate myself (@arjun024) as contributor

### DISCUSSION THREAD

**Discussion by arjun024** [paketo-buildpacks/feedback#152] [Stacks Team]
Hi, I would like to nominate myself as a contributor. I'm a maintainer on the Node, PHP, Python, Web, Utilities teams and would like to have privileges to contribute code directly/re-trigger CI jobs etc. You can see my contributions to the Paketo project at github.com/arjun024

@paketo-buildpacks/stacks-maintainers can I get a 👍 / 👎 ?

**Comment by arjun024** [comment]
@paketo-buildpacks/steering-committee Looks like I have the required votes. Could you add me to the stacks contrib team please?

**Comment by sophiewigmore** [comment]
It looks like you are a contributor now: https://github.com/orgs/paketo-buildpacks/teams/dependencies-contributors/members :) Congrats
